string Order_Id
string Item_type
